bioio_lif/reader.py

```diff
@@ -894,3 +897,84 @@ def get_mosaic_tile_positions(self, **kwargs: int) -> List[Tuple[int, int]]:
             )
 
         return adjusted_mosaic_positions
+
+    # --- Embedded metadata properties for the manifest ---
+    @staticmethod
+    def _convert_lif_timestamp(hex_timestamp: str) -> datetime.datetime:
+        """
+        Converts LIF timestamp from hexadecimal Windows FILETIME format to a UTC
+        datetime.
+
+        Example: '1db3d2d696a7a10' -> datetime(2024, 11, 22, 22, 25, 18, 897001)
```

Comment:
nit: for example date time, I like to have the minutes and seconds be over 31, so clearly not one of the other fields.